css/csscomb/readme.md

```diff
@@ -0,0 +1,36 @@
+# CSSComb
+
+A [CSS-beautifier utility](http://csscomb.com/) which will ensure that your SCSS is consistent with our [code style](https://github.com/mobify/mobify-code-style/blob/develop/css/.csscomb.json).
+
+![CSSComb in action](../../assets/csscomb.gif "CSSComb in action")
+
+## How to Install CSSComb
+
+### Prerequisites
+* Install Node.js and NPM with NVM as directed in the [Adaptive.js docs](http://adaptivejs.mobify.com/v2.0/docs/install/)
+* Install CSSComb and the Mobify Code Style:
+
+   `npm install -g csscomb mobify-code-style`
```

Comment:
Does csscomb have the ability to extend other configuration files? I understand ESlint does that, so that way they have a project level linter that extends our mobify-code-style linter.